source/wp-content/themes/wporg-news-2021/functions.php

```diff
@@ -389,3 +390,20 @@ function update_the_title( $title, $id ) {
 
 	return $title;
 }
+
+/**
+ * Remove "People of WordPress:" from the people post title.
+ *
+ * @param string $title The post title.
+ * @param int    $id    The post ID.
+ * @return string Filtered post title.
+ */
+function update_the_title_people( $title, $id ) {
+	// Remove "WordPress" from the post title in the Latest Release section on the front page.
+	$category_slugs = wp_list_pluck( get_the_category( $id ), 'slug' );
```

Comment:
```suggestion
```

You can get rid of this since you're not checking for it on the next line.

Reply:
Ah, I'm glad you noticed. I'm not entirely sure if this function is too generic — I just copy/pasted it from the function above.

Reply:
yeah, this could probably be re-refactored back into one `update_the_title` function that does both things.